include/arch/x86/cpuid.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2020 Amazon.com, Inc. or its affiliates.
```

Comment:
Should this be updated?

Reply:
The code has been copied as-is, hence I did not touch the copyright comment.